addons/mail/static/src/discuss/core/common/thread_model_patch.js

```diff
@@ -409,7 +429,7 @@ const threadPatch = {
      * @returns {import("models").ChannelMember[]}
      */
     get membersThatCanSeen() {
-        return this.channel_member_ids;
+        return this.channel.channel_member_ids ?? [];
```

Comment:
If channel is not guarded in this flow, fallback to array is not necessary. It's either both guard and fallback, or nothing special.